swarmzero_config_example.toml

```diff
@@ -1,3 +1,5 @@
+max_iterations = 10
```

Comment:
it also needs to be implemented for each agent i.e. `[target_agent_id]` should have `max_iterations` within it as well